openstates/nc/bills.py

```diff
@@ -70,10 +75,13 @@ def scrape_bill(self, chamber, session, bill_id):
             bill_type = "bill"
             bill_id = bill_id[0] + "B " + bill_id[1:]
 
-        bill_title = doc.xpath(
-            '/html/body/div/div/main/div[2]/div[contains(@class,"col-12")]/a'
-        )[0]
+        bill_title = doc.xpath("/html[1]/body[1]/div[1]/div[1]/main[1]/div[2]/div[1]")[
```

Comment:
can you elaborate on this change?  does this work for all session as-is?

Reply:
Not all of the bill titles are within a link. The x-path works for all sessions.